packages/api-client/src/components/Sidebar/Actions/SidebarListElementForm.vue

```diff
@@ -36,21 +25,21 @@ const variants = cva({
         Cancel
       </ScalarButton>
       <ScalarButton
-        class="custom-scroll whitespace-nowrap"
-        :class="cx(variants({ danger }))"
-        type="submit">
+        class="gap-1.5 font-medium px-2.5 h-8 shadow-none focus:outline-none custom-scroll whitespace-nowrap"
```

Comment:
so the theory behind the variants is we don't need to add other classes, we just have set variants and it would make them all uniform as well. But in practice that is never the case 😉

Reply:
wait i have no idea what's that 😂 wrong rebase

Reply:
👍🏾 no worries, my dream is to one day have an awesome, strict design system!

Reply:
yes size can usually be a prop

Reply:
wait size is already a prop?

Reply:
yes it is, but the sm for instance keep the height of the medium

Reply:
but i'll dig that up in that pr, i think we use small button for auth or request body, so we could switch them up to `xs` and then use small for `h-8` here?

Reply:
You don't need to do anything now, just keep it in mind! 🧠